kotskinds/apis/kots/v1beta1/helmchart_types.go

```diff
@@ -26,7 +26,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// +kubebuilder:validation:Type=""
```

Comment:
i think we need to keep this. vendors use this custom resource to define certain values to build airgap images and now they would be required to use this custom struct instead of it being any yaml key:value interface.